ConfigApp/Effects.cs

```diff
@@ -410,6 +410,7 @@ public enum EffectTimedType
             { "misc_fakeuturn", new EffectInfo("Fake U-Turn", EffectCategory.Misc) },
             { "misc_esp", new EffectInfo("ESP", EffectCategory.Misc, true) },
             { "screen_bouncyradar", new EffectInfo("Bouncy Radar", EffectCategory.Screen, true) },
+            { "cocktail_shaker", new EffectInfo("Cocktail Shaker", EffectCategory.Misc, true) },
```

Comment:
```suggestion
            { "cocktail_shaker", new EffectInfo("Cocktail Shaker", EffectCategory.Misc, true, true) },
```
Last flag is for whether the effect is short timed